gradle/detect.go

```diff
@@ -18,27 +18,28 @@ package gradle
 
 import (
 	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-
 	"github.com/buildpacks/libcnb"
 	"github.com/paketo-buildpacks/libpak"
 	"github.com/paketo-buildpacks/libpak/bard"
+	"os"
+	"path/filepath"
 )
 
 const (
 	PlanEntryGradle                = "gradle"
 	PlanEntryJVMApplicationPackage = "jvm-application-package"
 	PlanEntryJDK                   = "jdk"
 	PlanEntrySyft                  = "syft"
+	PlanEntryYarn				   = "yarn"
+	PlanEntryNode				   = "node"
 )
 
 type Detect struct{}
 
 func (Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error) {
 
-	l := bard.NewLogger(ioutil.Discard)
+	result := libcnb.DetectResult{}
+	l := bard.NewLogger(os.Stdout)
 	cr, err := libpak.NewConfigurationResolver(context.Buildpack, &l)
```

Comment:
```suggestion
	l := bard.NewLogger(os.Stdout)
	cr, err := libpak.NewConfigurationResolver(context.Buildpack, nil)
```

When you switch to writing to `os.Stdout` and pass that logger into NewConfigurationResolver my understanding is that it will lot to `os.Stdout` the configuration setting. They wouldn't normally show up but if you enabled the `pack build -v` flag they would show up and that's probably not necessary. I'm not sure that we want them to log during detect, it seems like extra noise to me. Do you think this would be helpful? 

If you set this to `nil` I'm pretty sure it'll skip logging them, but you can still log stuff later.